lms/templates/emails/enroll_email_allowedmessage.txt

```diff
@@ -18,16 +18,16 @@ ${_("To access the course visit {course_about_url} and register for the course."
 % endif
 % else:
 
-${_("To finish your registration, please visit {site_name}/dashboard\n"
-        "You will need to login with Touchstone to access this page. If you "
-        "do not have a Touchstone account, please contact your instructor for "
-        "detailed directions.").format(
+${_("To access the course, you will first need to register for MITx at {site_name}"
+        "with your Kerberos ID or Touchstone account."
+        "If you do not have a Kerberos ID and need to set up a Touchstone account,
```

Comment:
`or` instead of `and` ?

Reply:
My thinking was that there wouldn't be a case where someone wouldn't have a Kerberos ID and also wouldn't need to set up a Touchstone account.  Maybe it should just say, "If you need to set up a Touchstone account".